capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    super.capitalize
+  end
```

Comment:
- [ ] To make this method work you shouldn't call super however you should use the nameable instance variable inherited from the decorator call correct_name and use the capitalize method on that, something like:
```    
    def correct_name
      @nameable.correct_name.capitalize
    end
```